### PR TITLE
Feature - Image options

### PR DESCRIPTION
**Pull request by jjidt**
This is a tracker PR for image options/conversion/metadata work

### CODE REVIEW

File: workers/src/coordinators/imageProcessing.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import logger from '../libs/logger'
+import { LOG_LEVELS } from '../constants/logging'
+import portwayAPI from '../integrators/portwayAPI'
+
+sharp.cache(false)
+sharp.concurrency(0)
+
+const IMAGE_TEMP_DIRECTORY = process.env.IMAGE_TEMP_DIRECTORY || 'image_temp/'
+
+// TODO: update field with new image urls
```
Comment (jjidt) [inline workers/src/coordinators/imageProcessing.js]: this is handled, no longer TODO

File: workers/src/integrators/s3.js [hunk 2]
```diff
@@ -27,3 +27,25 @@ export const uploadExportZip = async function(filePath, uniqueId) {
...
+    Key: key,
+    ContentType: contentType,
+    Body: buffer,
+    ACL: 'public-read'
+  }
+
+  return s3.upload(params).promise()
+}
+
+// const promisifyStreamPipe = (readStream, writeStream) => {
```
Comment (jjidt) [inline workers/src/integrators/s3.js]: not used anymore, remove

**Comment by dheniges** [comment]
Reviewed the sharp processing and it looks fine with the buffers since streaming didn't really do anything, and it's synchronous per generated image. Just a couple TODOs you flagged JJ

**Comment by jjidt** [comment]
updated

**Review by dheniges — APPROVED** [review]
:shipit: :idt: :tada: :realjj: :sealeel:

**Pull request merged**